recipe-server/package.json

```diff
@@ -36,7 +36,7 @@
     "localforage": "1.4.3",
     "lodash.get": "4.4.2",
     "moment": "^2.19.3",
-    "mozilla-normandy-action-argument-schemas": "0.2.0",
+    "mozilla-normandy-action-argument-schemas": "0.3",
```

Comment:
It works!
<img width="956" alt="screen shot 2018-04-23 at 11 01 45 am" src="https://user-images.githubusercontent.com/26739/39134940-d7d44dd6-46e5-11e8-9dd9-478c4c027db3.png">